examples/Makefile

```diff
@@ -0,0 +1,16 @@
+# Build the rust test programs 
```

Comment:
Rather than a Makefile, this should be an additional target for `cargo build`. eg [cargo build --examples. ](https://doc.rust-lang.org/cargo/commands/cargo-build.html)

Reply:
I agree, this should be in `cargo build`